source/common/router/router.h

```diff
@@ -377,14 +419,41 @@ class Filter : Logger::Loggable<Logger::Id::router>,
     return cookie_value;
   }
 
+  void onUpstream100ContinueHeaders(Http::ResponseHeaderMapPtr&& headers,
```

Comment:
`// RouterFilterInterface` comment over these.

Do we care that the order of these in the .cc no longer follow the order in the .h file?

Reply:
I think preserving git history is more useful than having them in order, but happy to move them if others prefer the inverse.

Reply:
That's a good enough reason for me